src/zulip.rs

```diff
@@ -268,16 +283,56 @@ async fn query_pr_assignments(
                 .collect::<Vec<String>>()
                 .join(", ");
 
-            let review_prefs = get_review_prefs(&db_client, gh_id).await?;
+            let review_prefs = get_review_prefs(&db_client, gh_id)
+                .await
+                .context("cannot get review preferences")?;
             let capacity = match review_prefs.and_then(|p| p.max_assigned_prs) {
-                Some(max) => format!("{max}"),
+                Some(max) => max.to_string(),
                 None => String::from("Not set (i.e. unlimited)"),
             };
 
-            let mut response = format!("Assigned PRs: {prs}\n");
+            let mut response = format!("Assigned rust-lang/rust PRs: {prs}\n");
```

Comment:
Does the review assignment limit only work on rust-lang/rust? Shouldn't it work under the all repo?

Reply:
It currently only works for rust-lang/rust and is hardcoded for that single repo. I guess that eventually it should work for all/most repos, but the semantics are unclear (is the PR limit per repo or global?), and since we now load the workqueue eagerly, we would need to know which repos to load when triagebot starts, so we would need to hardcode the repo list in the source code, because triagebot doesn't currently know beforehand on which repos it is enabled (from its POV it's enabled everywhere).

So I would keep it only for rust-lang/rust for now, it's most useful there anyway. I haven't heard people wanting to use this on other repos (yet).